### PR TITLE
ability to use alternate bluetooth channel

### PR DESCRIPTION
**Pull request by efidoman**
This PR adds the ability to instruct Logger to use an alternate bluetooth channel to communicate to the Dexcom Transmitter. The alternate_bluetooth_channel setting can be set to true or false in the xdripjs.json settings file.

{
  "transmitter_id": "8XXXXX",
  "sensor_code": "1234",
  "mode": "not-expired",
  "pump_units": "mg/dl",
  **"alternate_bluetooth_channel": true,**
  "fake_meter_id": "000000"
}

The default if not specified is false.

The following are notes for when this parameter is set to true:
- The receiver cannot be used (the alternate bluetooth channel is the receiver's channel)
- One other system can be used (ie. XdripPlus or Dexcom Iphone app) alongside Logger
- Bluetooth collisions may be more frequent causing more missed readings due to "peripheral disconnected" issues. One recommendation is to keep the phone and the rig physically separated by a few inches (separate pockets) in order to reduce the chance of bluetooth interference.

Also this PR cleans up fallback mode to give actual tx state and sensor status (instead of Needs Calibration).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,6 +102,7 @@ You can edit the following options values in the configuration file (~/myopenaps
...
 	"pump_units" = Optional - pumpUnits default is "mg/dl"
 	"fake_meter_id" = Optional - meterid for fakemeter sending glucose records to pump, default is "000000"
+	"alternate_bluetooth_channel" = true or false. Optional - Default is false. If set to true then Logger uses the alternate channel to connect to the Dexcom transmitter. If set to true (the receiver cannot be used. However, when set to true, either the Xdip Plus android app or the Dexcom Iphone app can be used alongside Logger. Keep in mind that there is a higher chance for bluetooth conflict when connecting to the transmitter with both channels. You may be able to avoid some reconnects by keeping the rig and the phone app physically separated by several inches.
```
Comment (jpcunningh) [inline README.md]: This line has a unintended open parenthesis: `(`.

File: xdrip-get-entries.sh [hunk 2]
```diff
@@ -53,6 +53,14 @@ main()
     fi
   fi
 
+  if [ -z  "$alternateBluetoothChannel" ]; then
```
Comment (jpcunningh) [inline xdrip-get-entries.sh]: How would this ever not be an empty string?  Undocumented environment variable setting?

Comment (efidoman) [reply xdrip-get-entries.sh]: Good point. I will remove that check

File: README.md (hunk 1 above)
Comment (efidoman) [reply README.md]: Thanks - I'll correct it.

**Pull request merged**